common/djangoapps/student/views.py

```diff
@@ -350,6 +351,19 @@ def _cert_info(user, course, cert_status):
         else:
             status_dict['download_url'] = cert_status['download_url']
 
+            # getting linkedin URL and then pass the params which appears
+            # on user profile
+            linked_in_url = LinkedInUrlConfiguration.current().linkedin_url
+            cert_name = '{type} Certificate for {cert_name}'.format(
+                type=cert_status["mode"].title(), cert_name=course.display_name
+            )
+            params_dict = {
+                'pfCertificationName': cert_name,
+                'pfCertificationUrl': cert_status['download_url']
+            }
+
+            status_dict['linked_in_url'] = linked_in_url + urlencode(params_dict)
```

Comment:
I'd prefer if this code added the initial "&" so that we don't need to remember to add it to the configuration URL.